### DISCUSSION TITLE
How to set Select item style using item value

### DISCUSSION THREAD

**Discussion by rovashan** [mantinedev/mantine#4493] [Q&A] [answered]
I am trying to use the item value as the style property.

Example: There is a list of font family data.

`const fontFamilies = ['Arial', 'Courier New', 'Georgia', 'Gill Sans', 'Helvetica']`;

The Select is rendered using this data and I want to set the font family of each item based on its value in the data.

```
<Select
        placeholder="Font"
        nothingFound="No options"
        defaultValue="Arial"
        data={fontFamilies}
        styles={(theme) => ({
              item: {
                fontFamily: {item.value}
              }
        })}
/>
```

**Comment by rtivital** [comment, marked as answer]
Use custom item component – https://mantine.dev/core/select/#custom-item-component

**Reply by rovashan** [reply]
Brilliant! Thanks. Having lots of fun with your library.
